wpjquery

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+function wpjquery() {
+  ~/bin/migbin/wp plugin install enable-jquery-migrate-helper --skip-themes --skip-plugins
+  sleep 1 
+  ~/bin/migbin/wp plugin activate enable-jquery-migrate-helper --skip-themes --skip-plugins
+}
```

Comment:
You should use the --activate flag to make this a 1liner it's more efficient this way since it doesnt need 2 seperate wpcli calls.
```suggestion
function wpjquery() {
  ~/bin/migbin/wp plugin install enable-jquery-migrate-helper --activate --skip-themes --skip-plugins
}
```

Reply:
added flag